README.md

```diff
@@ -65,6 +65,24 @@ The Docker image won't build the UI until you run it, in order to be able to use
 variables and facilitate the deployment process. The env vars that you need to pass to the
 `docker run` cmd are the same as those in the `.env.example` file but without the `VITE` prefix.
 
+Example:
+
+```sh
+docker run \
+-e ETHEREUM_RPC_URL=http://localhost:8545 \
+-e ETHEREUM_EXPLORER_URL=https://goerli.etherscan.io \
+-e ETHEREUM_BRIDGE_CONTRACT_ADDRESS=0x0165878A594ca255338adfa4d48449f69242Eb8F \
+-e ETHEREUM_PROOF_OF_EFFICIENCY_CONTRACT_ADDRESS=0x8dA3b8020401851438eEe8bB434c57b54999935c \
+-e POLYGON_ZK_EVM_RPC_URL=http://localhost:8123 \
+-e POLYGON_ZK_EVM_EXPLORER_URL=http://localhost:4000 \
+-e POLYGON_ZK_EVM_BRIDGE_CONTRACT_ADDRESS=0x9d98deabc42dd696deb9e40b4f1cab7ddbf55988 \
+-e POLYGON_ZK_EVM_NETWORK_ID=1 \
+-e BRIDGE_API_URL=http://localhost:8080 \
+-e USE_FIAT_EXCHANGE_RATES=false \
+-e SHOW_OUTDATED_NETWORK_MODAL=false \
+-p 127.0.0.1:8080:80/tcp --name zkevm-bridge-ui -t zkevm-bridge-ui:local
```

Comment:
Why adding the `--name` and `-t` arguments? If we just want to run the image this could be omitted and should be up to each user to tweak them to their needs.

With the port, we should be able to just do: `-p 80:8080`.

Reply:
> Why adding the `--name` and `-t` arguments?

Well, it's just an example command, I think that the reachest the better. A user with little experience in docker can benefit from these example names and choose the ones he wants.

> With the port, we should be able to just do: `-p 80:8080`.

Yep, but that would [expose the UI to any machine in the network](https://docs.docker.com/engine/reference/commandline/run/#publish-or-expose-port--p---expose).

> Note that ports which are not bound to the host (i.e., `-p 80:80` instead of `-p 127.0.0.1:80:80`) will be accessible from the outside.

But sure, we can change this if you prefer it.

Reply:
> > Why adding the `--name` and `-t` arguments?
> 
> Well, it's just an example command, I think that the reachest the better. A user with little experience in docker can benefit from these example names and choose the ones he wants.
> 
> > With the port, we should be able to just do: `-p 80:8080`.
> 
> Yep, but if I understand this correctly that would [expose the UI to any machine in the network](https://docs.docker.com/engine/reference/commandline/run/#publish-or-expose-port--p---expose).
> 
> > Note that ports which are not bound to the host (i.e., `-p 80:80` instead of `-p 127.0.0.1:80:80`) will be accessible from the outside.
> 
> But sure, we can change this if you prefer it since it might be the desired behavior.

> A user with little experience in docker can benefit from these example names and choose the ones he wants.

Probably a user without experience won't even know what the `name` or `t` parameters mean. I think that we should provide just the minimal command to run the app. Probably a user who needs a name in the service won't even run it alone, he will run it inside a `docker-compose` or similar. And same with the `t` param, the app will be running but why would a regular user want to interact with the container through a terminal?

> Yep, but if I understand this correctly that would [expose the UI to any machine in the network](https://docs.docker.com/engine/reference/commandline/run/#publish-or-expose-port--p---expose).

Yep, in this case we would like to have access from the outside, as less experimented users won't be aware of this. Also, all docker images examples use this pattern, so I'd go with it.

Reply:
> why would a regular user want to interact with the container through a terminal?

https://docs.docker.com/engine/reference/run/#foreground

> -t: Allocate a pseudo-tty

🤦 now I get your comment! I thought the `-t` was to specify the image!

Reply:
Done!